var/spack/repos/builtin/packages/blt/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Blt(Package):
+    """BLT is a streamlined CMake-based foundation for Building, Linking and
+       Testing large-scale high performance computing (HPC) applications."""
+
+    homepage = "https://github.com/LLNL/blt"
+    url      = "https://github.com/LLNL/blt/archive/v0.3.6.tar.gz"
```

Comment:
The unit tests are failing because you have main and develop versions but not `git` to fetch them from.

Reply:
Fixed.